### PR TITLE
Improve grammar and flow

### PR DESCRIPTION
**Pull request by leila-alderman**
Improved the grammar and flow of Web Dev 101/Installations/ Prerequisites, Text Editors, and Installing Ruby.

### CODE REVIEW

**Comment by leila-alderman** [comment]
Sorry for the ugliness of this PR. My intent here was to include only the changes that I made to improve grammar and to exclude the changes I made to revise the section format (which are already included in another PR). Hopefully that's sufficiently clear and done correctly. 

Thanks!

**Comment by 105ron** [comment]
Looks good to me :+1:.

File: web_development_101/installations/installing_ruby.md [hunk 1]
```diff
@@ -78,58 +73,56 @@ echo 'eval "$(rbenv init -)"' >> ~/.bashrc
...
 
-Run this command within your terminal:
+Inside the terminal, run this command:
 
 ~~~bash
 rbenv install 2.5.3 --verbose
 ~~~
 
-This will take some time. We use the `--verbose` flag so you can see what it is doing and can be sure it hasn't gotten stuck. Take this time to watch [this video](https://www.youtube.com/watch?v=GzkfOKkIteA), or get a glass of water while it installs.
+This command will take 10-15 to complete. The `--verbose` flag will show you what's going on so you can be sure it hasn't gotten stuck. While it installs, take this time to watch [this video](https://www.youtube.com/watch?v=GzkfOKkIteA) or to get a glass of water.
```
Comment (105ron) [inline web_development_101/installations/installing_ruby.md]: 10-15 Could we add minutes here?

Comment (leila-alderman) [reply web_development_101/installations/installing_ruby.md]: Thanks for catching that!

**Comment by leila-alderman** [comment]
Got it! Thanks for catching that!

**Comment by 105ron** [comment]
Merged, thank you for your contribution. :smile:

**Pull request merged**